pkg/antctl/raw/multicluster/resourceimport.go

```diff
@@ -0,0 +1,187 @@
+		Args:    cobra.MaximumNArgs(1),
+		Example: resourceImportExample,
+		RunE:    runE,
+	}
+	o := &resourceImportOptions{}
+	options = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of ResourceImports")
+	command.Flags().StringVarP(&o.outputFormat, "output", "o", "", "Output format. Supported formats: json|yaml")
+	command.Flags().BoolVarP(&o.allNamespace, "all-namespace", "A", false, "If present, list ResourceImports across all namespaces")
+
+	GetCmd.AddCommand(command)
+}
+
+func runE(cmd *cobra.Command, args []string) error {
+	options.validateAndComplete()
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	kubeconfig.GroupVersion = &schema.GroupVersion{Group: "", Version: ""}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	scheme := k8sruntime.NewScheme()
+	err = mcsscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	err = antreamcscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	err = k8sscheme.AddToScheme(scheme)
+	if err != nil {
+		return err
+	}
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: scheme})
+	if err != nil {
+		return err
+	}
+
+	var res []multiclusterv1alpha1.ResourceImport
+	resourceImportList := &multiclusterv1alpha1.ResourceImportList{}
+	err = k8sClient.List(context.TODO(), resourceImportList, &client.ListOptions{Namespace: options.namespace})
+	if err != nil {
+		return err
+	}
+
+	argsNum := len(args)
+	for _, ri := range resourceImportList.Items {
+		if argsNum > 0 && ri.Name == args[0] {
+			res = append(res, ri)
+			break
+		} else if argsNum == 0 {
+			res = append(res, ri)
+		}
+	}
+	if len(res) == 0 {
+		if argsNum == 0 {
+			if options.namespace != "" {
+				fmt.Printf("No resource found in namespace %s\n", options.namespace)
+			} else {
+				fmt.Println("No resource found in all namespaces")
+			}
+			return nil
+		}
+		return fmt.Errorf("Error from server (NotFound): ResourceImport %s not found.", args[0])
+	}
+
+	switch options.outputFormat {
+	case "json":
+		byteJson, err := json.Marshal(res)
+		if err != nil {
+			return err
+		}
+		var prettyJson bytes.Buffer
+		err = json.Indent(&prettyJson, byteJson, "", " ")
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(prettyJson.String())
+	case "yaml":
+		yamlOutput, err := yaml.Marshal(res)
+		if err != nil {
+			return err
+		}
+		fmt.Println(string(yamlOutput))
+	default:
+		fmt.Print(output(res))
+	}
+
+	return nil
+}
+
+func output(resourceImports []multiclusterv1alpha1.ResourceImport) string {
+	var output strings.Builder
+	formatter := "%-25s%-100s%-25s\n"
+	output.Write([]byte(fmt.Sprintf(formatter, "NAMESPACE", "NAME", "KIND")))
+	sort.SliceStable(resourceImports, func(i, j int) bool {
+		return strings.Compare(resourceImports[i].Namespace, resourceImports[j].Namespace) == 1
+	})
+
+	for _, ri := range resourceImports {
+		fmt.Fprintf(&output, formatter, ri.Namespace, ri.Name, ri.Kind)
```

Comment:
the kind here shouldn't be `ri.Kind`, it's always `ResourceImport`, it's should be `re.Spec.Kind`

Reply:
done